engine/crates/integration-tests/tests/federation/basic/operations.rs

```diff
@@ -0,0 +1,166 @@
+use engine_v2::Engine;
+use graphql_mocks::FakeGithubSchema;
+use integration_tests::{engine_v1::GraphQlRequest, federation::EngineV2Ext, runtime};
+
+#[test]
+fn multiple_operations_without_name() {
+    let response = runtime().block_on(async move {
+        let engine = Engine::builder().with_subgraph(FakeGithubSchema).build().await;
+
+        engine
+            .post(
+                r"
+                    query First {
```

Comment:
I don't get this test, the operations have names here right?

Reply:
ah without providing an `operationName`